Backend/src/routes/api.js

```diff
@@ -126,25 +125,86 @@ function unpackData(data) {
         // Add bool to the front of dataArray
         dataArray.unshift(Boolean(data.readUInt8(buffOffset)));
         break;
+      case "uint8":
+        switch (property) {
+          case "tstamp_hr":
+            const hours = data.readUInt8(buffOffset);
+            if (hours < 10) hours = "0" + hours;
+            timestamps[0] = hours + timestamps[0];
```

Comment:
```suggestion
            if (hours < 10) timestamps[0] = "0" + hours + timestamps[0];
            else timestamps[0] = hours + timestamps[0];
```
Can't reassign a const